DigitalOffice/Services/CompanyService/appsettings.json

```diff
@@ -9,5 +9,9 @@
   "AllowedHosts": "*",
   "ConnectionStrings": {
     "SQLConnectionString": "Server=localhost;Database=CompanyDB;Trusted_Connection=True;"
+  },
+  "ServiceInfo": {
+    "ID": "47EC83DB-F697-4212-AA50-8D10EF1F17BC",
+    "Name": "CompanyService"
```

Comment:
I suggest using localhost ports as Ids.

Reply:
Whose localhost ports?

Reply:
this service

Reply:
Do you mean change "47EC83DB-F697-4212-AA50-8D10EF1F17BC" on localhost's port?

Reply:
Yes, example: 5016